### ISSUE TITLE
[Bug] [a11y] Show password is read, but cannot be accessed when creating an account

### ISSUE THREAD

**Issue by softvision-miralobontiu** [mozilla-mobile/fenix#7711]
## Steps to reproduce
1. Turn Talkback or Voice Assistant on.
2. Go to Settings - Turn on Sync and create a new account.
3. Fill in a valid email account, a valid password and then repeat the password.
4. Tap on the "Show password" icon.

### Expected behavior
The "Show password" icon is read and double-tapping on it is accessed and the user can see the password filled.

### Actual behavior
The "Show password" icon is read even if it isn't highlighted, but double-tapping on it does nothing. The password is not displayed, the user cannot see it.

### Device information
* Android device: Samsung Galaxy Tab S3 (Android 8)
* Fenix version: Beta 3.1.0-beta 3

![Untitled](https://user-images.githubusercontent.com/40694158/72424722-1a574780-378f-11ea-9fb0-0f5c0924e2ea.png)

**Comment by sblatz** [comment]
This is again an issue with FxA's side. I will contact them about it
